Neurips-contribution (#70)
SwissDataScienceCenter/renku-blog: blog/2024-04-18-contribution-neurips/index.mdx

---
slug: neurips-2023
title: |

>   Renku in NeurIPS: a platform for sustainable data science

Reviewer: ```suggestion
  Renku at NeurIPS: presenting a platform for sustainable data science
```

authors: [elisabet, rokroskar, jimena]
tags:
  - renku
  - citation
  - poster
  - neurips
image: image_1.jpg
---

![Renku in NeurIPS 2023](./image_1.jpg)

<!-- truncate -->

The Swiss Data Science Center presented Renku in the [37th Conference
on Neural Information Processing Systems (NeurIPS)](https://neurips.cc/Conferences/2023),
which took place on Dec. 10th to 16th 2023 in New Orleans (US).

> The contribution “Renku: a platform for sustainable data science” presents

Reviewer: I think it looks nice to have the paper title stand out as a clickable link
```suggestion
Our contribution [“Renku: a platform for sustainable data science”](https://papers.neurips.cc/paper_files/paper/2023/file/838694e9ab6b0a193b84daaafcac0eed-Paper-Datasets_and_Benchmarks.pdf) presents
```

those Renku features that make it a unique platform to support sustainable
development and use of data and code, and showcases how Renku works in
different scientific domains.

> You can access our [work](https://papers.neurips.cc/paper_files/paper/2023/file/838694e9ab6b0a193b84daaafcac0eed-Paper-Datasets_and_Benchmarks.pdf)
> and [complementary material](https://proceedings.neurips.cc/paper_files/paper/2023/file/838694e9ab6b0a193b84daaafcac0eed-Supplemental-Datasets_and_Benchmarks.pdf) online.
> 
> Please, check our poster and presentation [here](https://neurips.cc/virtual/2023/poster/73464)

Reviewer: I like how you have the paper and complementary materials highlighted as the links so that they stand out- shall we do the same for the poster and presentation? 

Also- "work" seems like a weird word to use to refer to the paper, but maybe you know better than I do?

Also- "Please, ..." strikes me a too formal. There's something about the comma that makes it sound like a command to me, and not a encouragement :P

I included all three points above in my suggestion below, but I may have broken formatting trying to edit it in the browser 😬:
```suggestion
Check out our [paper (https://papers.neurips.cc/paper_files/paper/2023/file/838694e9ab6b0a193b84daaafcac0eed-Paper-Datasets_and_Benchmarks.pdf), 
[complementary materials](https://proceedings.neurips.cc/paper_files/paper/2023/file/838694e9ab6b0a193b84daaafcac0eed-Supplemental-Datasets_and_Benchmarks.pdf),
and [poster and presentation](https://neurips.cc/virtual/2023/poster/73464).
```

The Swiss Data Science Center presented Renku in the [37th Conference
on Neural Information Processing Systems (NeurIPS)](https://neurips.cc/Conferences/2023),
which took place on Dec. 10th to 16th 2023 in New Orleans (US).
Our contribution [“Renku: a platform for sustainable data science”](https://papers.neurips.cc/paper_files/paper/2023/file/838694e9ab6b0a193b84daaafcac0eed-Paper-Datasets_and_Benchmarks.pdf) presents
those Renku features that make it a unique platform to support sustainable
development and use of data and code, and showcases how Renku works in
different scientific domains.

> Check out our [paper (https://papers.neurips.cc/paper_files/paper/2023/file/838694e9ab6b0a193b84daaafcac0eed-Paper-Datasets_and_Benchmarks.pdf),

Reviewer: I messed up the formatting in my suggestion and link isn't rendering correctly - sorry!
```suggestion
Check out our [paper](https://papers.neurips.cc/paper_files/paper/2023/file/838694e9ab6b0a193b84daaafcac0eed-Paper-Datasets_and_Benchmarks.pdf),
```
